server/sqlstore/playbook_run.go

```diff
@@ -475,6 +475,21 @@ func (s *playbookRunStore) FinishPlaybookRun(playbookRunID string, endAt int64)
 	return nil
 }
 
+func (s *playbookRunStore) RestorePlaybookRun(playbookRunID string, restoredAt int64) error {
+	if _, err := s.store.execBuilder(s.store.db, sq.
+		Update("IR_Incident").
+		SetMap(map[string]interface{}{
+			"CurrentStatus":      app.StatusInProgress,
+			"EndAt":              0,
+			"LastStatusUpdateAt": restoredAt,
+		}).
+		Where(sq.Eq{"ID": playbookRunID})); err != nil {
```

Comment:
nit: could we wrap this down one line?
```suggestion
		Where(sq.Eq{"ID": playbookRunID})
	); err != nil {
```